components/src/operator/FetchInsertionsOperator.ts

```diff
@@ -2,28 +2,20 @@ import { type Dataset } from './Dataset';
 import { type Operator } from './Operator';
 import { fetchInsertions } from '../lapisApi/lapisApi';
 import { type InsertionEntry, type LapisFilter, type SequenceType } from '../types';
-import { MutationCache } from '../utils/mutations';
+import { Insertion } from '../utils/mutations';
 
 export class FetchInsertionsOperator implements Operator<InsertionEntry> {
     constructor(
         private filter: LapisFilter,
         private sequenceType: SequenceType,
     ) {}
 
-    private async fetchInsertions(
-        lapis: string,
-        signal?: AbortSignal,
-    ): Promise<{ insertion: string; count: number }[]> {
-        return (await fetchInsertions(lapis, { ...this.filter }, this.sequenceType, signal)).data;
-    }
-
     async evaluate(lapisUrl: string, signal?: AbortSignal): Promise<Dataset<InsertionEntry>> {
-        const insertions = await this.fetchInsertions(lapisUrl, signal);
+        const insertions = (await fetchInsertions(lapisUrl, { ...this.filter }, this.sequenceType, signal)).data;
```

Comment:
Why do we need the destructuring here?
```suggestion
        const insertions = (await fetchInsertions(lapisUrl, this.filter, this.sequenceType, signal)).data;
```